qcodes/instrument_drivers/Gentec/Gentec_Maestro.py

```diff
@@ -0,0 +1,66 @@
+from qcodes import VisaInstrument
+from qcodes.utils.helpers import create_on_off_val_mapping
+
+
+class Gentec_Maestro(VisaInstrument):
+    """
+    Instrument driver for the Gentec Maestro powermeter.
+    Args:
+        name (str): Instrument name.
+        address (str): ASRL address
+        baud_rate (int): Baud rate for the connection.
+    Attributes:
+        model (str): Model identification.
+        firmware_version (str): Firmware version.
+        
+    """
+
+    def __init__(self, name, address, baud_rate=115200, **kwargs):
+
+        super().__init__(name, address, **kwargs)
+
+        # set baud rate
+        self.visa_handle.baud_rate = baud_rate
+
+        # get instrument information
+        self.model = self.visa_handle.query('*VER').split()[0]
+        self.firmware_version = self.visa_handle.query('*VER').split()[2]
+
+        # add parameters
+        self.add_parameter('power',
+                           get_cmd='*CVU',
+                           get_parser=float,
+                           label='power',
+                           unit='W')
+
+        def wavelength_get_parser(ans):
+            return int(ans[4:])
+        self.add_parameter('wavelength',
+                           get_cmd='*GWL',
+                           set_cmd='*PWC{0:0>5}',
+                           get_parser=wavelength_get_parser,
+                           label='wavelength',
+                           unit='nm')
+
+        def zero_offset_get_parser(ans):
+            return int(ans[5:])
+        self.add_parameter('zero_offset_status',
```

Comment:
would `zero_offset_enabled` be a better name?

Reply:
I did change the name of the parameter.